charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -463,9 +463,9 @@ redis:
 #Turning either hearings api (with either listener on) or cft hearing service on requires
 #service bus to be turned on.
 sscs-hearings-api:
-  enabled: false
+  enabled: true
   java:
-    image: "hmctspublic.azurecr.io/sscs/hearings-api:latest" #Change to pr if required
+    image: "hmctspublic.azurecr.io/sscs/hearings-api:427" #Change to pr if required
```

Comment:
What is the need for this, and is it the cause of the build failure?

Reply:
The hearing changes are not merged into Master  yet..
So required to use the PR version for the testing..